Vault-Deployment/helm-install.tf

```diff
@@ -63,11 +63,13 @@ resource "kubernetes_secret" "prometheus-monitoring-token" {
 
 
 resource "helm_release" "prometheus-grafana" {
-  name       = "kube-stack-prometheus"
-  repository = "https://prometheus-community.github.io/helm-charts"
-  chart      = "kube-prometheus-stack"
-  namespace  = var.namespace
-  version    = "44.2.1"
+  name             = "kube-stack-prometheus"
+  repository       = "https://prometheus-community.github.io/helm-charts"
+  chart            = "kube-prometheus-stack"
+  create_namespace = true
+  namespace        = "kube-prometheus-stack"
+  version          = "55.0.0"
+  wait             = false
```

Comment:
@Xelef2000 On my machine adding the `wait` flag here as well (not waiting) performed better in many ways